10LessonTen/Checkpoint2/index.html

```diff
@@ -1,10 +1,81 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+  <title>Checkpoint #2</title>
+  <link rel="stylesheet" href="css/css-reset.css" />
+  <link rel="stylesheet" href="css/style.css" />
 </head>
 <body>
 
+  <div class="panel" id="navigation">
+    <h3>Company/Organization</h3>
+    <nav>
+      <a href="#">Link</a>
+      <a href="#">Link</a>
+      <a href="#">Link</a>
+      <a href="#">Link</a>
+      <a href="#">Link</a>
+    </nav>
+  </div>
+  <main>
+    <div class="panel">
```

Comment:
I know I said I don't care about content that much, but next time please use some color and image that you find. It's good practice to use images that don't quite fit because that's what you'll be given from clients.